grakn-graph/src/test/java/ai/grakn/generator/AbstractTypeGenerator.java

```diff
@@ -92,11 +98,20 @@ private final boolean includeNonMeta() {
         return !meta.orElse(false);
     }
 
+    private final boolean includeAbstract(){
+        return isAbstract.orElse(false);
+    }
```

Comment:
This should be `.orElse(true)`. Essentially, if you don't specifically ask to include/exclude abstract types, then they should be included.

I'm wondering if there's a way to remove the `Optional` from `isAbstract` and `meta`, but I'm not sure...